training/intel-bootc/Containerfile

```diff
@@ -1,62 +1,58 @@
+ARG DRIVER_TOOLKIT_IMAGE="quay.io/ai-lab/nvidia-builder:latest"
```

Comment:
```suggestion
ARG DRIVER_TOOLKIT_IMAGE="quay.io/ai-lab/intel-builder:latest"
```

Reply:
I was under the assumption this had to do with what he explained in the comment, where we will be starting with the `nvidia-builder` image name, and then would do a follow on PR to rename it to just `builder` as "it is currently named after NVIDIA, but should be renamed in a follow-up change. The benefit is that a single builder image is maintained for all bootc images that require out-of-tree drivers."

Reply:
Even if I prefer a builder for each driver I would change the name of the builder to something like `common-builder` to avoid confusion.

Reply:
Does that image exist today? I mention that it should become a unique image not related to a vendor, but in a follow PR.

Reply:
To clarify this nothing currently exists in quay, weve only been using local builds of dtk